src/Action/ExceptionAction.php

```diff
@@ -80,6 +83,10 @@ public function __invoke(FlattenException $exception, Request $request): Respons
         $headers['X-Content-Type-Options'] = 'nosniff';
         $headers['X-Frame-Options'] = 'deny';
 
+        if ($exception->getStatusCode() == Response::HTTP_INTERNAL_SERVER_ERROR && !$this->debug) {
```

Comment:
Please use `===` and Yoda style. I would also invert the order because `!$this->debug` is cheaper.

`if (!$this->debug && Response::HTTP_INTERNAL_SERVER_ERROR === $exception->getStatusCode() ) {`